### PR TITLE
Try to deploy observe-gn-dev to heroku

### PR DESCRIPTION
**Pull request by cquiroz**

### CODE REVIEW

**Review by jluhrs — APPROVED** [review]
LGTM

File: deploy/src/main/resources/conf/app.conf [hunk 1]
```diff
@@ -6,7 +6,7 @@
 
 # If mode is DEVELOPMENT or STAGING, UI provides some extra tools
 environment = STAGING
-site = GS
+site = ${?SITE}
```
Comment (rpiaggio) [inline deploy/src/main/resources/conf/app.conf:9]: I think that if you use `${SITE}` (without the `?`) the app will fail in the absence of the env variable. Maybe that's what we want.

File: deploy/src/main/resources/conf/app.conf [hunk 2]
```diff
@@ -19,7 +19,7 @@ smart-gcal {
...
...pD/a7O3QQ38T\nufvV15ARtizcAmAEglOJmzC/f8PjygyImt3QGfIzdv+JXQp8Ww06lnI7rij3M/gk\nJQBoxHrUszbdhE/E5JvqudNSy4KNFFH2gnyx55QmqHis3Cai6tbXvjpIFGzI1reA\nsxYKC2iEIl3jVv9qLZPVYju6vFAaFL4vPbi9OQf7yHjLZ97eqO4VANg162xYO+sr\n3az0rywQS6C3cl8oTXC3Fx7M2UvSVjQjif/ji7Rcg53prB3Cj60conawiQARAQAB\ntCNMdWN1bWEgU1NPIDxyb2Iubm9ycmlzQG5vaXJsYWIuZWR1PokCVwQTAQgAQRYh\nBHyvVN9WDKRIVlSY35f9xbLdAQ96BQJkNEMjAhsDBQkDwmcABQsJCAcCAiICBhUK\nCQgLAgQWAgMBAh4HAheAAAoJEJf9xbLdAQ96aYwP/0OgEaehsGdwjB+AGYR6k897\ntVsgd6vWlQDAysGwVtUFT8ZpjDWy90qKyd7Syj8N36v70XpeCtXuX1GieftL5Vna\nRdpncWlhTl4oMAH/5/ZLCSsVTE/cLjRCR0V5hjow2tbG2a+XOX85eSTIrNuarVfz\n1jCqY01cGbS7+OJLdgGcoF+xDLISkJxzKIZcTfMIdNXcAPRzNjPQ3v0DiUfUUALD\neKFcKPdAtsEr5GT7OzRh4PrOzS41lfLV0Zo68TA/POWUDJKw/p2eROs0K5utj3BP\nP0/OB1xSns1oyBK1iV5NS327DnvhtmVwJZ3ypKe5x0xfwBoEIv7+GNrS2B1aT5Bt\nPUmWPaMxwrz11ok6DOi8dZLMPzzHelXgOlxZ4aqZdi/+EMWAb0MupDNk4Tqp3LO4\ny6bzY/J+SlrOFueoXKmFL7u9Y/3NMKS/u8PzHUvR1mRowbKBx86oBNNcDpaUqWdE\nfeq0loPI2eiayic+gSWFSdIwEgjfs2yN4WwCR+Wed9PBm18/7kh3zn8CA+vivWJV\nwCUwNi8gwTP2BEfAB801n8v327tW9aQVv74WmnbDLaO3nepupHn+Vt2SLnUwaOGN\nKQeP3PSZxqczHCnyn++TI2sVDa07EZgQg1VshfuE+BVTLa72D73CZ9zK04zbFlBm\nmyXfbbs0K6ZHIj9KIVk3\n=jo/I\n-----END PGP PUBLIC KEY BLOCK-----"
```
Comment (rpiaggio) [inline deploy/src/main/resources/conf/app.conf:23]: Doesn't the public key change when we change the server?

Comment (cquiroz) [reply deploy/src/main/resources/conf/app.conf:23]: Good question, I was able to login though

File: deploy/src/main/resources/conf/app.conf (hunk 1 above)
Comment (cquiroz) [reply deploy/src/main/resources/conf/app.conf:9]: Right we do want that

**Pull request merged**